Darkplaces-Emscripten/builddeps/buildgmp.sh

```diff
@@ -0,0 +1,16 @@
+if ! type "emcc" > /dev/null; then
+    printf "\nEmscripten Not installed. Please install Emscripten at https://emscripten.org/docs/getting_started/downloads.html \n"
+    exit 1
+fi
+#m4 and libtool are required to compile. 
+
+wget https://gmplib.org/download/gmp/gmp-6.3.0.tar.xz
```

Comment:
This also appears to be available as a package install
```
libgmp-dev/testing,now 2:6.3.0+dfsg-2 amd64 [installed,automatic]
  Multiprecision arithmetic library developers tools
```

Reply:
Same problem, needs to be compiled for WASM